ledger/accountdb.go

```diff
@@ -1102,13 +1121,44 @@ func writeCatchpointStagingCreatable(ctx context.Context, tx *sql.Tx, bals []nor
 	return nil
 }
 
+// writeCatchpointStagingKVs inserts all the KVs in the provided array into the
+// catchpoint kvstore staging table catchpointkvstore, and their hashes to the pending
+func writeCatchpointStagingKVs(ctx context.Context, tx *sql.Tx, kvrs []encodedKVRecordV6) error {
+	insertKV, err := tx.PrepareContext(ctx, "INSERT INTO catchpointkvstore(key, value) VALUES(?, ?)")
+	if err != nil {
+		return err
+	}
+	defer insertKV.Close()
+
+	insertHash, err := tx.PrepareContext(ctx, "INSERT INTO catchpointpendinghashes(data) VALUES(?)")
```

Comment:
I guess I was wondering about "tablespace" locks vs page locks. But
>However, since there is only one WAL file, there can only be one writer at a time.

So it probably does not matter if have multiple writers to the same table or different ones

Reply:
and the subsequent question - would it benefit from a fan-in writer

Reply:
Resolving - Based on group discussion we decided as is behavior is acceptable.

Reply:
I suppose that, strictly speaking, we need to drop support for any prefix of the form 0xFF*. That is, any prefix of 0xff bytes (zero or more).